www/src/record/record.list.html

```diff
@@ -1,4 +1,4 @@
-<ion-view view-title="Registros">
+<ion-view view-title="{{ vm.research.name }}">
```

Comment:
why would I use **$ionicNavBarDelegate**? I agree with one-binding and prefer with `{{ ::vm.research.name }}` because you need to load the research entity before loading the screen, so it already solves this before you enter the screen, preventing screen is loaded if the `id` is invalid.